### DISCUSSION TITLE
Server crashes repeatedly when a player joins the game

### DISCUSSION THREAD

**Discussion by alessandrolomanto** [wolveix/satisfactory-server#373] [General]
Hi everyone,

I've been noticing that the server crashes repeatedly, even when just one person joins the game.

Here are the server specs:

VPS on Contabo

- 8 CPU cores
- 24GB RAM
- 1.2TB SSD

Using Grafana, I observed some unusual CPU spikes. When these occur, the player currently online gets disconnected, losing their character and inventory. This player becomes permanently offline, and upon logging back in, a new character is created.

At first, I thought the issue might be CPU-related, but now I’m wondering if it could be an I/O problem instead.

For context:

![Screenshot 2024-11-29 at 09 22 58](https://github.com/user-attachments/assets/c9b0ca05-8db9-4d31-8add-02141867ddfa)
![Screenshot 2024-11-29 at 09 29 07](https://github.com/user-attachments/assets/2adb71eb-f2ce-4af7-8ffc-e89be8daa640)
![Screenshot 2024-11-29 at 09 29 37](https://github.com/user-attachments/assets/baa91038-56f6-4c0f-b64e-7da1f2b2b66c)
![Screenshot 2024-11-29 at 09 33 24](https://github.com/user-attachments/assets/26ca1809-0624-4a66-91e2-eae295a3981a)

I'm using Docker Compose with satisfactory-server:latest.
Grafana stack is also part of the setup.
I've attached some Grafana screenshots for reference.

Have you experienced the same issue? How did you solve it?

**Comment by wolveix** [comment]
Hey @alessandrolomanto, this sounds like a game bug. Could you please provide the debug output of the container/fill out the normal issue template please? What CPU does your VPS use?
